src/main/resources/config/application.properties

```diff
@@ -62,11 +62,13 @@ docmosis.uri=${DOCMOSIS_SERVICE_BASE_URL:https://docmosis-development.platform.h
 documents.english.DIRECTION_ISSUED=TB-SCS-GNO-ENG-00091.docx
 documents.english.DECISION_ISSUED=TB-SCS-GNO-ENG-00091.docx
 benefit_specific_documents.pip.english.ISSUE_FINAL_DECISION=TB-SCS-GNO-ENG-00642.docx
+benefit_specific_documents.gen.english.ISSUE_FINAL_DECISION=TB-SCS-GNO-ENG-00642.docx
```

Comment:
Do you think there will ever be a time where these docs will be different? Seems like a good time to combine all of this to clear up the config?

Reply:
As discussed, going to implement in a separate PR